packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -253,14 +253,27 @@ export class PendingStateManager implements IDisposable {
 		}
 	}
 
+	public processPendingLocalBatch(
+		batch: InboundSequencedContainerRuntimeMessage[],
+		batchStartCsn: number,
+	): {
+		message: InboundSequencedContainerRuntimeMessage;
+		localOpMetadata: unknown;
+	}[] {
+		return batch.map((message) => ({
```

Comment:
Future: we'll be able to move `maybeProcessBatchStart` and `maybeProcessBatchEnd` to the start/end of this function an drop the `maybe` :)